ntbs-service/Pages/Notifications/Edit/_SaveButton.cshtml

```diff
@@ -6,6 +6,7 @@
 @{
     var isDraft = Model.NotificationStatus == NotificationStatus.Draft;
```

Comment:
As a general point, I do think the way this view is written is quite confusing, so feel free to refactor it a bit if you want